src/status_im/ui/components/large_header.cljs

```diff
@@ -0,0 +1,149 @@
+(ns status-im.ui.components.large-header
+  (:require-macros [status-im.utils.views :refer [defview letsubs]])
+  (:require [reagent.core :as reagent]
+            [cljs-bean.core :refer [->clj ->js]]
+            [status-im.ui.components.colors :as colors]
+            [status-im.ui.components.list.views :as list.views]
+            [status-im.ui.components.react :as react]
+            [status-im.ui.components.toolbar.view :as toolbar]
+            [status-im.ui.components.toolbar.styles :as toolbar.styles]
+            [status-im.utils.platform :as platform]
+            [status-im.ui.components.animation :as animation]))
+
+(def hidden (reagent/atom 0))
+(def shown (reagent/atom 100))
+(def header-profile-visible? (reagent/atom false))
+(def initial-on-show-done? (volatile! false))
+
+(defview my-profile-toolbar-content [header-in-toolbar has-nav? show?]
```

Comment:
TODO probably remove "profile" in this naming, as this namespace is for the purpose of a generic component